modules/olm-rukpak-about.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-packaging-format.adoc
+
+:_content-type: CONCEPT
+[id="olm-rukpak-about_{context}"]
+= RukPak (Technology Preview)
+
+:FeatureName: RukPak
+include::snippets/technology-preview.adoc[]
+
+{product-title} 4.12 introduces the _platform Operator_ type as a Technology Preview feature. The platform Operator mechanism relies on the RukPak component, also introduced in {product-title} 4.12, and its resources to manage content.
+
+RukPak consists of a series of controllers, known as _provisioners_, that install and manage content on a Kubernetes cluster. RukPak also provides two primary APIs, `Bundle` and `BundleDeployment`. These components work together to bring content onto the cluster and install it, generating resources within the cluster.
+
+A provisioner places a watch on both `Bundle` and `BundleDeployment` resources that refer to the provisioner explicitly. For a given bundle, the provisioner unpacks the contents of the `Bundle` resource onto the cluster. Then, given a `BundleDeployment` resource referring to that bundle, the provisioner installs the bundle contents and is responsible for managing the lifecycle of those resources.
+
+Two provisioners are currently implemented and bundled with RukPak: the _plain provisioner_ sources and unpacks `plain+v0` bundles, and the _registry provisioner_ sources and unpacks Operator Lifecycle Manager (OLM) `registry+v1` bundles.
```

Comment:
I had to re-read this to understand it, because I first read it as "the plain provisioner source does something..." Maybe add for clarity:

```suggestion
Two provisioners are currently implemented and bundled with RukPak: the _plain provisioner_ that sources and unpacks `plain+v0` bundles, and the _registry provisioner_ that sources and unpacks Operator Lifecycle Manager (OLM) `registry+v1` bundles.
```